common/src/main/scala/org/apache/comet/CometConf.scala

```diff
@@ -307,6 +307,18 @@ object CometConf extends ShimCometConf {
       .booleanConf
       .createWithDefault(false)
 
+  val COMET_EXEC_SHUFFLE_WITH_HASH_PARTITIONING_ENABLED: ConfigEntry[Boolean] =
```

Comment:
Can a user have both configs enabled? What happens?

Reply:
The default is both enabled. They individually control whether hash or range partitioning falls back, respectively.

Reply:
That's what I thought. Is there a way to add a unit test with both enabled?